spec/acceptance/manage/bzr_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'vcsrepo::manage::bzr' do
```

Comment:
I read somewhere that the `if:` part is deprecated and will be removed in RSpec 4, but not 100% sure.

It's also confusing that `os` comes from serverspec / specinfra and `os[:family]` is more like the OS name. So Ubuntu doesn't fall under the Debian OS family. That will cause it to silently ignore it. Perhaps it's better to use `unless: ...` because then it'll at least fail.

Reply:
Thanks for the help with my mess of ruby!